.github/workflows/build.yml

```diff
@@ -15,6 +15,37 @@ concurrency:
 
 jobs:
 
+  init:
+    runs-on: ubuntu-latest
+    outputs:
+      python_versions: ${{ steps.generate-matrix.outputs.python_versions }}
+      build_windows: ${{ steps.generate-matrix.outputs.build_windows }}
+      test_windows: ${{ steps.generate-matrix.outputs.test_windows }}
+      build_macos: ${{ steps.generate-matrix.outputs.build_macos }}
+      test_macos: ${{ steps.generate-matrix.outputs.test_macos }}
+      build_linux: ${{ steps.generate-matrix.outputs.build_linux }}
+      test_linux: ${{ steps.generate-matrix.outputs.test_linux }}
+    steps:
+      - name: Generate Matrix
+        id: generate-matrix
+        run: |
+          echo "Set Matrix"
+          echo "::set-output name=python_versions::['3.7', '3.8', '3.9']"
+          echo "::set-output name=build_windows::['windows-2016']"
+          echo "::set-output name=test_windows::['windows-latest']"
+          echo "::set-output name=build_macos::['macos-latest']"
+          echo "::set-output name=test_macos::['macos-10.15', 'macos-11']"
+          echo "::set-output name=build_linux::['ubuntu-latest']"
+          echo "::set-output name=test_linux::['ubuntu-18.04', 'ubuntu-20.04']"
+
+  my_echo:
```

Comment:
Please remove this job; do not keep debugging stuff in production code, this is useless and may add more bugs

Reply:
done